### ISSUE TITLE
Automatically open newly created Work Items

### ISSUE THREAD

**Issue by quenumat** [forcedotcom/devops-center-roadmap#108]
Today, when a new Work Item is created, we have to scroll all the way down to the end of the view to open it and include items to deploy.

It would be efficient to provide a toast notification with the link to the new Work Item allowing us to open it easily, or to allow us to create our own list view that can be pinned to help us open newly created Work Items first, or to allow us to get re-directed automatically to the newly created Work Item.

**Comment by naveensimplify** [comment]
Also reorder the workitems with created date order by descending. So that we will see the most recently created workitems always on top (even if we logout and login to the system again).

**Comment by petrov-allupp** [comment]
I agree with the comment of @naveensimplify , and would like to add the idea to let us configure default filter options when we access the Workitems tab. For example, i don't find any benefit of seeing the Closed Workitems by default and would like to see those which are in progress. Maybe also automatically fill in the Assigned To filter with the current running user. This combination would allow me to quickly see what my current in progress Workitems are.
